src/data/real/nnreal.lean

```diff
@@ -80,6 +80,12 @@ iff.intro nnreal.eq (congr_arg coe)
 lemma ne_iff {x y : ℝ≥0} : (x : ℝ) ≠ (y : ℝ) ↔ x ≠ y :=
 not_iff_not_of_iff $ nnreal.eq_iff
 
+protected lemma «forall» {p : ℝ≥0 → Prop} : (∀ x : ℝ≥0, p x) ↔ ∀ (x : ℝ) (hx : 0 ≤ x), p ⟨x, hx⟩ :=
```

Comment:
Could you please do this for all `nonneg` types? See the file `algebra.order.nonneg`.

Reply:
This will not make these lemmas available for `nnreal`, because `nnreal` is not a notation/abbreviation. So, I suggest that we add type-specific aliases like `nnreal.forall` whenever we need them first. I need `nnreal.forall` for #13675

Reply:
Ok